package.json

```diff
@@ -3,11 +3,11 @@
     "version": "3.0.0",
     "private": true,
     "scripts": {
```

Comment:
Can you exclude this file from the commit?

Reply:
yeah sure! mybad haha